packages/amplify-graphql-transformer-interfaces/src/transform-host-provider.ts

```diff
@@ -0,0 +1,70 @@
+import {Duration, Stack} from '@aws-cdk/core';
```

Comment:
```suggestion
import { Duration, Stack } from '@aws-cdk/core';
```

There are a few more of these below.